src/components/homePage/index.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import backgroundLogo from '../assets/bg_logo.svg'
+import logo from '../assets/logo.svg'
+import styled from '@emotion/styled'
+
+const HomePage = () => {
```

Comment:
There isn't really a good place for this comment, so it'll be against this line.

Since this is a page - or a route if you prefer - then this shouldn't be a component. This kind of ties in to my earlier question of "Why not just edit the actual home page"?

Reply:
I agree with you, but I didn't touched the existing components as it's the first time I'm contributing here so wasn't aware of any old conversations regarding the components that are created so rather preferred to create a separate one. If you suggest I can refactor this component and add my code in the existing `home` component. Please advice @nobrayner

Reply:
Please do :smile:

Reply:
Pushed updated changes